index.html

```diff
@@ -1,87 +1,183 @@
+                        </ul>
+                    </section>
+                    <section class="skills-1">
+                        <h5 class="skills-2">SKILLS</h5>
+                        <ul class="skills-3">
+                            <li>Codekit</li>
+                            <li>Gitdub</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </section>
+                </div>
+            </div>
+        </main>
         <div class="icons-1">
-         <div class="arrow-left"><img src="images/arrow.png" alt="arrow-down"></div>
-            <ul class="icons-2">
-                <li><img src="images/Disabled.png" alt="github-icon"></li>
-                <li><img src="images/Frame.png" alt="LinkedIn-icon"></li>
-                <li><img src="images/angelist.png" alt="angellist-icon"></li>
-                <li><img src="images/Group.png" alt="twitter-icon"></li>
-                <li><img src="images/medium.png" alt="Medium icon"></li>
+            <div class="work-left">
+                <img class="arrow-image left-arrow" src="images/arrow.png" alt="arrow-down">
+            </div>
+            <ul class="links">
+                <li><a href="#"><img src="images/github.png" alt="github-icon"></a></li>
+                <li><a href="#"><img src="images/linkedin.png" alt="linkedIn-icon"></a></li>
+                <li><a href="#"><img src="images/angellist.png" alt="angellist-icon"></a></li>
+                <li><a href="#"><img src="images/twitter.png" alt="twitter-icon"></a></li>
+                <li><a href="#"><img src="images/medium.png" alt="medium-icon"></a> </li>
             </ul>
-            <div><img src="images/arrow.png" alt="arrow-down"></div>
-        </div>
\ No newline at end of file
+            <div class="work-right">
+                <img class="arrow-image right-arrow" src="images/arrow.png" alt="arrow-down">
+            </div>
+        </div>
+        <section class="container-4">
+            <h1 class="title-1">
+                my recent work
+            </h1>
+            <section class="project p-1">
+                <div class="picture">
+                   <img src="images/to-do-list.jpg" alt="to-do-image">
+                </div>
+              <div class="project-body">
+                <h2 class="title-2">TO-DO-LIST</h2>
+                 <p>Add, Prioritize & Organize your Team's Tasks! Be more efficient and successful! Start with. 
+                    A simple To-Do-List, add more Details as you go ! 
+                   Free for up to 50 User! Try it now! Instant Messaging. 
+                   Intuitive Drag & Drop. Up-To Date Information.
+                  </p>
+                  <ul class="skills-4">
+                    <li>CSS</li>
+                    <li>HTML</li>
+                    <li>JavaScript</li>
+                  </ul>
```

Comment:
- [ ] The font is not matched with the Figma design. Please input the correct font family for this one and the rest of them also.